winui/Scheduler/Appointment-Drag-And-Drop.md

```diff
@@ -11,6 +11,8 @@ documentation: ug
 
 The Scheduler supports to reschedule the appointment by performing the drag and drop operation.
 
+N> In Drag and Drop functionality is break due to the latest Microsoft.WindowsAppSDK version 1.1.
+
 ## Disable drag and drop
 
```

Comment:
Update the same in Known issue in release notes also

Reference - https://help.syncfusion.com/winui/release-notes/v19.1.0.54?type=all#known-issues